docs/conf.py

```diff
@@ -228,6 +228,7 @@
     "sunkit_instruments": ("https://docs.sunpy.org/projects/sunkit-instruments/en/stable/", None),
     "zeep": ("https://docs.python-zeep.org/en/stable/", None),
     "contourpy": ("https://contourpy.readthedocs.io/en/stable/", None),
+    "itables": ("https://mwouts.github.io/itables/quick_start.html", None),
```

Comment:
```suggestion
    "itables": ("https://mwouts.github.io/itables/objects.inv", None),
```
This might work, not sure.

Reply:
Still not working, should I revert it back?

Reply:
Yeah